src/components/attendee/AttendeeLayout.module.scss

```
@@ -5,7 +5,7 @@
 @use "scss/attendee/layer";
 @use "scss/attendee/breakpoint";
 
-$-z-index--space: layer.above(0);
+$-z-index--space: layer.$z-index--chatbox;
```

This is the same as 
```suggestion
$-z-index--space: layer.level(layer.$z-index--chatbox);
```

but also, can just use `layer.$z-index--chatbox` instead of `$-z-index--space` everywhere it is needed.

And if the chatbox name isn't as suitable, maybe a new name common for both components on the same layer can be found.

---

[ce03ad0](https://github.com/sparkletown/sparkle/pull/2973/commits/ce03ad0c063b92163c5039c84f70f34635278d0b)

---

added `$z-index--space` constant for this component 
I think it's better to have an equal z-index value, but with a corresponding name for this exact component, so we don't mix up multiple in the future
[624bf0a](https://github.com/sparkletown/sparkle/pull/2973/commits/624bf0ae08d4a3cad7c2160cc459785c715e4c79)